lib/service.js

```diff
@@ -93,26 +88,31 @@ class Service {
     });
   }
 
+  static _checFormData(formData) {
+    if(isEmpty(formData)) {
+      throw new Error('Can\'t be empty');
+    }
+
+    if (!isObject(formData)) {
+      throw new Error('Should be an object');
+    }
+  }
```

Comment:
But what a proper values for formData, in formData we can have any types o fields, what exactly I should check?